doc/classes/SceneTree.xml

```diff
@@ -198,7 +198,7 @@
 			<param index="1" name="property" type="String" />
 			<param index="2" name="value" type="Variant" />
 			<description>
-				Sets the given [param property] to [param value] on all nodes inside this tree added to the given [param group]. Nodes that do not have the [param property] are ignored. See also [method call_group] and [method notify_group].
+				Calls [method Object.set] to set the given [param property] to [param value] on all nodes inside this tree added to the given [param group]. Nodes that do not have the [param property] are ignored. See also [method call_group] and [method notify_group].
```

Comment:
I don't think the internal implementation details should be exposed. It doesn't make any difference how the value is set.

Reply:
1. You have that more information about Object calls already provided in other sections, take a look.  Is your plan to remove those later because of this general sentiment that it makes the Reference hard to understand?  My take was that a Reference doc SHOULD have those kinds of details, but tutorials and other doc types, maybe not.

2. In the case of the Godot notifications URL, do you agree that's useful?  My partner and I spent 2 hours figuring out what was meant generally by "notifications", which is a very loose term that isn't covered in the manual where it's needed with a hyperlink, but instead only the word "notifications".

Reply:
@thadguidry I think it is fair to treat `Object.set` differently from `Object.notification`. The docs for `Object.set` don't add any information that isn't already contained in `set_group`, so the link just makes the description seem more complex. 

Contrast that with `Object.notification` which adds a lot of information above what should be provided by `notify_group`. 

In general I agree with you that reference docs should be free to include more implementation details. But I do agree with the other reviewers here that in this case it is not helpful. 

> In the case of the Godot notifications URL, do you agree that's useful? My partner and I spent 2 hours figuring out what was meant generally by "notifications" and finally found the page and section. "Notifications" is a very loose term that isn't explained well enough in the manual when/where it's needed with hyperlinks, but instead only the word "notifications". So my PR also includes that change in SceneTree doc for that

I think this is a very useful change. Since it provides a link to a much more comprehensive doc that explains the concept of notifications much better than we can provide in a reference doc

Reply:
@clayjohn Understood, and see the point made about Object.set now.  Agree it doesn't add much value. (it just does the set...who cares how that's done, except for Godot devs themselves, I guess.)

OK, I'll remove the Object.set link and change, and only include the Godot notifcation URL change.

Reply:
Thank you!